pkg/proto/datadog/sbom/workloadmeta_sbom.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package datadog.sbom;
+
+option go_package = "pkg/proto/pbgo/sbom"; // golang
+
+message SBOMStreamParams {
+}
+
+message SBOMMessage {
+    bytes Data = 1;
+    string Kind = 2;
+    // ID is the ID for this entity, in a format specific to the entity Kind.
+    string ID = 3;
```

Comment:
Should we rename it to KindID to avoid confusion in the future?

Reply:
Shamelessly stole it from https://github.com/DataDog/datadog-agent/blob/main/comp/core/workloadmeta/def/types.go#L227-L228